### PR TITLE
Adding information about .less files presidence

### PR DESCRIPTION
**Pull request by zanilee**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will clarify Magento themes default behavior. Without it people think it should work in the different way: magento/magento2#17012

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=2591) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=2591) before we can accept your contribution.<br/>

**Comment by zanilee** [comment]
This information need to be added to clarify Magento behavior. Without it people think it should work in the different way: https://github.com/magento/magento2/pull/17012

File: guides/v2.1/frontend-dev-guide/css-guide/css_quick_guide_approach.md [hunk 1]
```diff
@@ -34,6 +33,10 @@ To extend the parent theme's styles in your theme:
 
 Extending a theme using <code>_extend.less</code> is the simplest option when you are happy with everything the parent theme has, but want to add more styles.
 
+<div class="bs-callout bs-callout-info" id="info-extend-priority">
```
Comment (shrielenee) [inline guides/v2.1/frontend-dev-guide/css-guide/css_quick_guide_approach.md]: We don't use HTML like this for callouts, or notes. Check out our Contributor guide for info on how to format callouts: https://devdocs.magento.com/guides/v2.2/contributor-guide/templates/basic_template.html.

Comment (shrielenee) [reply guides/v2.1/frontend-dev-guide/css-guide/css_quick_guide_approach.md]: Also, use markdown ALWAYS when you can.

**Comment by shrielenee** [comment]
Please sign the CLA. We won't be able to merge until that is signed! Thanks!

**Comment by zanilee** [comment]
Hi, @shrielenee!

The html replaced to a Callout Message.

Thanks

**Pull request merged**